src/docker/era_5g_netapp_base_mmcv/Dockerfile.gpu

```diff
@@ -0,0 +1,29 @@
+FROM nvidia/cuda:11.7.1-base-ubuntu20.04
+
+RUN apt-get update \ 
+    && apt-get install -y python3-pip git wget
+
+# GPU version
+RUN pip install torch torchvision torchaudio --extra-index-url https://download.pytorch.org/whl/cu117
```

Comment:
Pin versions? Otherwise, there might be a new `torch` for instance and things may break...

Reply:
Yes, that is a good idea.

Reply:
Version pinned for both cpu and gpu variants